paper-input-behavior.html

```diff
@@ -224,6 +224,13 @@
       },
 
       /**
+       * Bind this to the `<input is="iron-input">`'s `oninvalid` property.
+       */
+      oninvalid: {
+        type: String
+      },
+
+      /**
        * Limits the numeric or date-time increments.
        * Bind this to the `<input is="iron-input">`'s `step` property.
        */
```

Comment:
Yesss, but let me test that for a bit since it's a fairly big change. In any case, I think this PR can be closed :)